lib/motors/vector_controller.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "core/types.hpp"
+
+namespace hyped::motors {
+class VectorController {
+ private:
+  core::Float Ia;
+  core::Float Ib;
+  core::Float Ic;
+  core::Float rotorVelocity;
+  core::Float Id;
+  core::Float Iq;
+  core::Float Vd;
+  core::Float Vq;
+  core::Float IdRef;
+  core::Float IqRef;
+  core::Float Kp;
+  core::Float Ki;
+  core::Float theta;
```

Comment:
1. Rename most of these to be clear on what they actually are
2. Variables should be named in `lower_snake_case` and class data members should have a trailing `_`, so `current_phase_a_` etc.
3. Move the private block after public.